### DISCUSSION TITLE
UADE plugin

### DISCUSSION THREAD

**Discussion by Brog33** [emoon/HippoPlayer#212] [Q&A]
I got HippoPlayer running on my Mac, but the UADE plugin seems to be missing from the app (it's in the compatibility list). Is there a way to get it working?

**Comment by emoon** [comment]
So it seems that I fixed the compilation errors it was having at a point, but never did a new release with the plugin included. I'm trying to get it to build again (on Github) and publish a new (untested) version.

**Comment by emoon** [comment]
Can you give https://github.com/HippoPlayer/HippoPlayer/releases/tag/0.2.4 a try? It should include the UADE plugin, but notice that it's fully untested.

**Reply by Brog33** [reply]
Fantastic! It works :) It's now playing 'dw' and 'cust' files as a champ!! Thank you!
